.github/workflows/discord-congrats.yml

```diff
@@ -0,0 +1,15 @@
+name: Discord Notification on Push
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  congrats:
+    name: congratsbot
+    if: ${{ github.repository_owner == 'withastro' }}
+    uses: withastro/automation/.github/workflows/congratsbot.yml@main
+    with:
+      EMOJIS: '🎉,🎊,🧑‍🚀,🥳,🙌,🚀,<:houston_product_hunt:1145760743208263904>,<:notworsethanwhatwehadbefore:1020455379353751573> '
```

Comment:
I feel like this could be even more `<:houston...` emoji and maybe 🤖 for fun? (Picking the emoji is my favourite thing about this action 😁)

Reply:
Yeah. I was looking around and couldn't find any really robotic Houstons except product hunt

Reply:
I just added a few more that I felt kind of fit the vibe